dashboard/test/ui/features/assignment_on_overview_pages.feature

```diff
@@ -20,7 +19,7 @@ Feature: (Un)Assign on script and course overview pages
     And I press the first ".uitest-unassign-button" element
     # One assign button for each of 9 currently unassigned units in this course,
     # and one at the top of the page indicating the course is assigned
-    And I wait for the page to fully load
+    And I wait for 3 seconds
```

Comment:
please correct me if I am mistaken, but it looks like we'll be waiting for this server call to complete before the page is updated: https://github.com/code-dot-org/code-dot-org/blob/4b5bfbeb1f77070a7c5d23ce6455526c9258c300/apps/src/templates/teacherDashboard/teacherSectionsRedux.js#L237-L247

This makes me think that we'll likely see a recurrence of flakiness in this test. A couple ways to address this:
1. (recommended) wait until the first SectionAssigner no longer contains an unassign button, using this test step or something like it: https://github.com/code-dot-org/code-dot-org/blob/bc43830ade99a5583d58b2978e9d09f20a1366a8/dashboard/test/ui/features/step_definitions/steps.rb#L190
2. have the redux actions set some state when the server request completes, like we do here: https://github.com/code-dot-org/code-dot-org/blob/1cad5ac667f9ed1a3aa31063e6d06bd21a78e2bd/apps/src/code-studio/hiddenStageRedux.js#L183